### ISSUE TITLE
Inconsistent identification of volume and VMT difference in links in the Scenario Comparison tool

### ISSUE THREAD

**Issue by Flatiron87** [Triangle-Modeling-and-Analytics/TRMG2#426]
When the scenario comparison tool runs, it does not map all of the links with differences.  For example, when comparing the base_2020 to the 2050 (v1.3.2), the Total_Flow_Daily Differences.map shows gaps and zero differences, however there are differences. In the first figure, the black numbers are from the road layer in the map: Roads.ID, RoadName, Total_Flow_Daily, Total_Flow_Daily_ref, Total_Flow_new, Total_Flow_Daily_Diff, abs_Total_Flow_Daily_Diff.  The blue value is the 2020 total flow.  Note that link 24937 is highlighted correctly.  The adjacent link (163437 has a total 2050 flow of 29051 (same as adjacent link) and a base_2020 flow of 13,009 so it should have calcualted a difference (and show highlighted), however the difference is showing as 0. The VMT difference map is similar (see second figure)

![image](https://github.com/user-attachments/assets/612a0863-9be0-4b59-ad61-50189c589c9f)
![image](https://github.com/user-attachments/assets/2fd3fd6d-2538-4015-82a9-66dde934574e)

**Comment by dkyleward** [comment]
CC: @lfhuntsinger 

I ran fresh base and 2055 scenarios using the dev branch version of the model. Everything looks good. 

<img width="262" alt="image" src="https://github.com/user-attachments/assets/e186ed04-28af-4cbf-b6dd-217e129e0edc">

It may have been something we have fixed in the meantime, or it may be that the scenarios you are comparing aren't from the same version of the model (e.g. 2050 might have been run months ago while base is fresh).

Either way, the next release of the model will work as it should. Closing this issue as complete.
